easyjsonform.js

```diff
@@ -1043,6 +1073,7 @@ class EasyJsonForm {
         "item.properties.mandatory": "Mandatory",
         "item.number": "Number",
         "item.text": "Text",
+        "item.date": "Date",
```

Comment:
You may want to move this up a line to keep all of the `item.text` entries together.